docs/_includes/anchor_links.html

```diff
@@ -0,0 +1,27 @@
+<script>
```

Comment:
nit: Can we have a comment explaining what the code does? (my understanding is that it makes all headers links to themselves?)

Reply:
Will do, and yes, it makes the h3 elements anchor links as those are what we care about for the API documentation.

Reply:
done